01_stratified_classification/nbconverted/lasso_range_gene_optimizers.py

```diff
@@ -620,22 +627,19 @@ def bins_to_coords(bins):
     )
     g.axes[0].set_xlabel('Bin (increasing number of nonzero coefficients)')
     g.axes[0].set_xlim((0, perf_coefs_df.nz_quantile.max()))
+    g.axes[0].set_ylim((-0.05, 1.05))
     g.axes[1].set_xlabel('Bin (increasing number of nonzero coefficients)')
     g.axes[1].set_xlim((0, perf_coefs_df.nz_quantile.max()))
+    g.axes[1].set_ylim((-0.05, 1.05))
     g.set_ylabels(f'{metric.upper()}')
     sns.move_legend(g, "center", bbox_to_anchor=[1.045, 0.55], frameon=True)
     g._legend.set_title('Dataset')
     new_labels = ['train', 'holdout', 'test']
     for t, l in zip(g._legend.texts, new_labels):
         t.set_text(l)
-    g.set_titles('Optimizer: {col_name}')
-     
-    plt.suptitle(
-        f'Number of nonzero coefficients vs. {metric.upper()}, decile binning, {plot_gene}',
-        y=1.0
-    )
+    g.set_titles('Gene: {}, optimizer: {{col_name}}; decile binning'.format(plot_gene), y=1.05)
```

Comment:
I think you should have either all commas or all semicolons separating the parts.